docusaurus/docs/Flutter/guides/customize_message_widget.mdx

```diff
@@ -11,16 +11,16 @@ Customizing Text Messages
 Every application provides a unique look and feel to their own messaging interface including and not
 limited to fonts, colors, and shapes.
 
-This guide details how to customize the `MessageWidget` in the Stream Chat Flutter UI SDK.
+This guide details how to customize the `StreamMessageWidget` in the Stream Chat Flutter UI SDK.
 
 ### Building Custom Messages
 
-This guide goes into detail about the ability to customize the `MessageWidget`. However, if you want
-to customize the default `MessageWidget` in the `MessageListView` provided, you can use the `.copyWith()` method
-provided inside the `messageBuilder` parameter of the `MessageListView` like this:
+This guide goes into detail about the ability to customize the `StreamMessageWidget`. However, if you want
+to customize the default `StreamMessageWidget` in the `StreamMessageListView` provided, you can use the `.copyWith()` method
+provided inside the `messageBuilder` parameter of the `StreamMessageListView` like this:
 
 ```dart
-MessageListView(
+StreamMessageListView(
     messageBuilder: (context, details, messageList, defaultImpl) {
         // Your implementation of the message here
         // E.g: return Text(details.message.text ?? '');
```

Comment:
Maybe this existed from earlier versions, but since we mention the `.copyWith()` method, should we also show that in the following example?